Tests/TripKitUITests/TKUIAutocompletionRaceTest.swift

```diff
@@ -131,6 +132,31 @@ class TKUIAutocompletionRaceTest: XCTestCase {
       .completed(6),
     ]))
   }
+  
+  func testStableRace() throws {
+    let inputs: [([TKAutocompletionResult], at: TestTime)] = [
+      ([TKAutocompletionResult(object: "1", title: "Campbell High School", image: .iconAlert, score: 10)], at: 100),
+      ([TKAutocompletionResult(object: "2", title: "Highway 17", image: .iconAlert, score: 20)], at: 200),
+      ([TKAutocompletionResult(object: "3", title: "Campbell High School", image: .iconAlert, score: 15)], at: 300),
+      ([TKAutocompletionResult(object: "4", title: "Campbell High School Athletics", image: .iconAlert, score: 25)], at: 400),
+      ([TKAutocompletionResult(object: "5", title: "Campbell High School", image: .iconAlert, score: 5)], at: 500),
+      ([TKAutocompletionResult(object: "6", title: "Campbell HS - Basketball Courts", image: .iconAlert, score: 30)], at: 600)
+    ]
+    
+    let expected: [Recorded<Event<[TKAutocompletionResult]>>] = [
+      .next(600, [
+        TKAutocompletionResult(object: "6", title: "Campbell HS - Basketball Courts", image: .iconAlert, score: 30),
+        TKAutocompletionResult(object: "4", title: "Campbell High School Athletics", image: .iconAlert, score: 25),
+        TKAutocompletionResult(object: "2", title: "Highway 17", image: .iconAlert, score: 20),
+        TKAutocompletionResult(object: "5", title: "Campbell High School", image: .iconAlert, score: 5)
+      ]),
```

Comment:
Why does this keep 5? Shouldn't it be keeping the highest scoring one, i.e., 3 with a score of 15?

Reply:
Oh right, just placed in random scores for testing. I adjusted the lev threshold after this so this one fails, updating.